src/webapi/console.rs

```diff
@@ -0,0 +1,75 @@
+#[doc(hidden)]
+#[macro_export]
+macro_rules! console_unsafe {
```

Comment:
Could you rename this macro to something that's even more less likely to clash? (I don't know, `__console_internal`, or something like that?)

Reply:
Done.